mvc/tutorials/mvc-with-entity-framework/sample/src/ContosoBooks/Models/SampleData.cs

```diff
@@ -10,7 +10,8 @@ public static class SampleData
         public static void Initialize(IServiceProvider serviceProvider)
         {
             var context = serviceProvider.GetService<BookContext>();
-            if (context.Database.AsRelational().Exists())
+            
+            if (context.Database.EnsureCreated())
```

Comment:
I wouldn't do this, the tutorial correctly uses Migrations to create the database (since that will end up with an application that can be deployed and then updated in the future). If the migrations have already been applied then this will return false and you won't get any sample data. If the migrations have not been applied then this will create the database outside of migrations and you won't be able to add migrations in the future.

If you just want to check that the database exists then use:
```db.GetService<IRelationalDatabaseCreator>().Exists()```

Although maybe it is ok to assume it exists since the rest of the data access code in the application does as well. If the database doesn't exist then you'll skip this code but just hit a "could not connect" error later on when one of the actions tries to access the database